flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCInputFormatTest.java

```diff
@@ -101,6 +105,33 @@ public void testIncompleteConfiguration() throws IOException {
 	}
 
 	@Test
+	public void defaultFetchSizeIsUsedIfNotConfiguredOtherwise() throws SQLException {
+		jdbcInputFormat = JDBCInputFormat.buildJDBCInputFormat()
+			.setDrivername(DRIVER_CLASS)
+			.setDBUrl(DB_URL)
+			.setQuery(SELECT_ALL_BOOKS)
+			.setRowTypeInfo(ROW_TYPE_INFO)
+			.finish();
+		jdbcInputFormat.openInputFormat();
+		assertThat(jdbcInputFormat.getStatement().getFetchSize(), equalTo(1));
+
+	}
+
+	@Test
+	public void fetchSizeCanBeConfigured() throws SQLException {
+		final int desiredFetchSize = 10_000;
+		jdbcInputFormat = JDBCInputFormat.buildJDBCInputFormat()
+			.setDrivername(DRIVER_CLASS)
+			.setDBUrl(DB_URL)
+			.setQuery(SELECT_ALL_BOOKS)
+			.setRowTypeInfo(ROW_TYPE_INFO)
+			.setFetchSize(desiredFetchSize)
+			.finish();
+		jdbcInputFormat.openInputFormat();
+		assertThat(jdbcInputFormat.getStatement().getFetchSize(), equalTo(desiredFetchSize));
```

Comment:
Would be good to replace these with junit assertEquals, to be consistent with the remaining tests in flink.